editor/inspector/contributions/node.js

```diff
@@ -385,11 +385,15 @@ const Elements = {
     panel: {
         ready() {
             const panel = this;
-            let animationId;
+            let animationId = undefined;
 
             panel.__nodeChanged__ = (uuid) => {
-                if (Array.isArray(panel.uuidList) && panel.uuidList.includes(uuid)) {
+                if (typeof animationId === 'number') {
                     window.cancelAnimationFrame(animationId);
+                    animationId = undefined;
+                }
+
+                if (Array.isArray(panel.uuidList) && panel.uuidList.includes(uuid)) {
```

Comment:
This cannot be changed here, which will cause the update function to be interrupted unexpectedly